src/intl/Core__Intl__Collator.res

```diff
@@ -25,16 +25,10 @@ type resolvedOptions = {
 
 type supportedLocalesOptions = {localeMatcher: Core__Intl__Common.localeMatcher}
 
-@new external make: unit => t = "Intl.Collator"
-@new external makeWithLocale: string => t = "Intl.Collator"
-@new external makeWithLocales: array<string> => t = "Intl.Collator"
-@new external makeWithLocaleAndOptions: (string, options) => t = "Intl.Collator"
-@new external makeWithLocalesAndOptions: (array<string>, options) => t = "Intl.Collator"
-@new external makeWithOptions: (@as(json`undefined`) _, options) => t = "Intl.Collator"
-
-@val external supportedLocalesOf: array<string> => t = "Intl.Collator.supportedLocalesOf"
+@new external make: (~locales: array<string>=?, ~options: options=?) => t = "Intl.Collator"
```

Comment:
An alternative here could be:

```suggestion
@new external make: (array<string>, ~options: options=?) => t = "Intl.Collator"
```

as passing an empty array is effectively the same  This would be more convenient if the common case is  to pass one or more locales, but less so if it the default is more common. Not sure which is though.